api/BUILD

```diff
@@ -1,7 +1,53 @@
 load("//bazel:api_build_system.bzl", "api_proto_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_grpc_library")
 
 licenses(["notice"])  # Apache 2
 
+proto_library(
+    name = "api",
+    srcs = [
```

Comment:
Can you do something like https://github.com/envoyproxy/data-plane-api/pull/205/files#diff-bd16c6e30811a8577dbc96b795c1ba93R120 here? We need such a rule for docs generation, but I'm wondering if we can avoid duplicating the actual source specification and just use deps from the existing `api_proto_library` targets.

Reply:
Hm, doesn't seem to work. Empty srcs crashes something in rules_go or bazel. A smaller list of srcs does not generated `pb.go` files that I need.